k8s/aat/ccd/user-profile-api/user-profile-api.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccd-user-profile-api
```

Comment:
Isn't it better to align the names with file name ?

Reply:
not sure, happy with how it is for now